org.mfek.glif.yml

```
@@ -0,0 +1,44 @@
+app-id: org.mfek.glif
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```

```suggestion
runtime-version: '23.08'
```
23.08 is the latest runtime